src/lib/services/index.ts

```diff
@@ -268,10 +268,11 @@ export const createServices = (
     );
 
     const dependentFeaturesService = db
-        ? createDependentFeaturesService(db, config)
+        ? createDependentFeaturesService(config)(db)
         : createFakeDependentFeaturesService(config);
-    const transactionalDependentFeaturesService = (txDb: Knex.Transaction) =>
-        createDependentFeaturesService(txDb, config);
+    const transactionalDependentFeaturesService = db
```

Comment:
Do you need both dependentFeatureService and transactionalDependentFeaturesService? I think you can just 
```typescript
const dependentFeaturesService = transactionalDependentFeaturesService
```

Reply:
you're right, we don't need both.